Lab4.py

```diff
@@ -152,7 +148,7 @@ def temp_kanger(t, t_kanger):
 plt.colorbar(label='Temperature (°C)')
 plt.show()
 
-def plot_kanger(xmax=100, tmax=1825, dx=0.1, dt=0.2):
+def plot_kanger(xmax=100, tmax=1825, dx=0.1, dt=0.2, kanger_cc=0):
```

Comment:
Great change with kanger_cc! Makes the plotting more efficient in my code, which I really needed help with.